services/brig/index/src/Brig/Index/Options.hs

```diff
@@ -35,6 +36,9 @@ data Command
   = Create ElasticSettings
   | Reset ElasticSettings
   | Reindex ElasticSettings CassandraSettings
+  | ReindexSameOrNewer ElasticSettings CassandraSettings
+  | UpdateMapping (URIRef Absolute) ES.IndexName
```

Comment:
```suggestion
  | UpdateMapping (URIRef Absolute) ES.IndexName  -- (`ElasticSettings` is used here in brig/index; the function to update mapping lives in brig and can't use that.)
```

Reply:
I don't get it, can you please explain?

Reply:
sorry, i was not reading enough of the context.